.github/workflows/build-and-test.yaml

```diff
@@ -23,8 +38,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -r requirements.txt -r requirements-dev.txt
-          python setup.py develop
+          make install-dev
```

Comment:
Does this also need to run: "$ make install" to install the requirements.txt file as well

Reply:
ditto my other reply